telethon/events/callbackquery.py

```diff
@@ -26,9 +26,13 @@ class CallbackQuery(EventBuilder):
             can use ``re.compile(b'data_')``.
```

Comment:
`pattern` needs to be documented too. Not even I know what it's supposed to do.

Reply:
will add that.